js/webrtc.js

```diff
@@ -76,15 +76,19 @@ var spreedPeerConnectionTable = [];
 			return;
 		}
 
-		if (!webrtc.webrtc.getPeers(sessionId, 'screen').length) {
-			if (useMcu) {
-				// TODO(jojo): Already create peer object to avoid duplicate offers.
-				// TODO(jojo): We should use "requestOffer" as with regular
-				// audio/video peers. Not possible right now as there is no way
-				// for clients to know that screensharing is active and an offer
-				// from the MCU should be requested.
-				webrtc.connection.sendOffer(sessionId, "screen");
-			} else {
+		var screenPeers = webrtc.webrtc.getPeers(sessionId, 'screen');
+		if (useMcu && !screenPeers.length) {
+			// TODO(jojo): Already create peer object to avoid duplicate offers.
+			// TODO(jojo): We should use "requestOffer" as with regular
+			// audio/video peers. Not possible right now as there is no way
+			// for clients to know that screensharing is active and an offer
+			// from the MCU should be requested.
+			webrtc.connection.sendOffer(sessionId, "screen");
+		} else if (!useMcu) {
+			var ownScreenPeer = screenPeers.find(function(screenPeer) {
```

Comment:
this conflicts with the global var from line 17?
```
/drone/src/github.com/nextcloud/spreed/js/webrtc.js
  55:8   error  'ownScreenPeer' was used before it was defined  no-use-before-define
  56:5   error  'ownScreenPeer' was used before it was defined  no-use-before-define
  60:4   error  'ownScreenPeer' was used before it was defined  no-use-before-define
  71:31  error  'ownScreenPeer' was used before it was defined  no-use-before-define
72:4 error 'ownScreenPeer' was used before it was defined no-use-before-define
```

Reply:
Yes, it conflicts; please note the `2. developing` label, this fix is not finished yet.